osu.Game/Database/IModelDownloader.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Game.Online.API;
+using System;
+
+namespace osu.Game.Database
+{
+    public interface IModelDownloader<TModel> : IModelManager<TModel>
```

Comment:
(class) xmldoc should be placed on interface instead of abstract class, if we decide to go in this direction.